frontend/src/components/HospitalMap.tsx

```diff
@@ -17,7 +17,27 @@ class HospitalMap extends React.Component {
     lng: -0.09,
     zoom: 13,
   }
-
+  componentDidMount() {
+    /**
+     * Issue: #12
+     * Title: Detect user's location and use it as a search parameter
+     */
+    if('geolocation' in navigator){ /*Check if browser supports Geolocation API */
+      navigator.geolocation.getCurrentPosition((position) => {
+        console.log(position.coords)
+        /**
+         * Update state with latest Latitude and longitudes
```

Comment:
And for single-line comments, definitely use `//`. :)

Reply:
ok got it